src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
```

Comment:
`save()` is deprecated since [Hibernate 6.0](https://docs.jboss.org/hibernate/orm/6.0/javadocs/org/hibernate/Session.html#save(java.lang.Object)). Use `presist()` instead. 
Make sure to do it everywhere.